### ISSUE TITLE
downgrades version to @latest

### ISSUE THREAD

**Issue by gr2m** [peerigon/updtr#21]
When my repository is using a version higher than `@latest`, it downgrades to whatever is `@latest`. But if the currently installed version is higher than `@latest`, it shouldn't do anything

**Comment by vkeusebio** [comment]
@gr2m
run `$ updtr --exclude module1,modul2,...` if you don't want to update certain modules.

**Comment by jhnns** [comment]
> When my repository is using a version higher than @latest

Which versions can be higher than `@latest?` How are they formatted?

**Comment by gr2m** [comment]
@jhnns try this:

```
$ npm info semantic-release@latest version
4.3.5
$ npm info semantic-release@next version
6.0.3
``` 

`@latest` is the default distribution tag when you publish a package, but you can publish with custom tags, e.g. to publish beta releases of your software, without breaking all the packages out there. 

This is a good read for that context:
https://medium.com/greenkeeper-blog/one-simple-trick-for-javascript-package-maintainers-to-avoid-breaking-their-user-s-software-and-to-6edf06dc5617

**Comment by jhnns** [comment]
@gr2m Awesome read, thank you. I knew that this was possible, but I had not the occasion to experiment with it so far.

Yes, we should take this into account.

**Comment by MoOx** [comment]
I just got the same issue. I am using babel-eslint@6.0.0-beta.6 and it downgraded to babel-eslint@5.0.0 :'(

**Comment by MoOx** [comment]
The good part is: it failed to do so, so it didn't kept the change :)

**Comment by matthaias** [comment]
@gr2m @MoOx @leecade

It should be solved now with `v0.1.14`. Sorry for the late fix! Please check it and if there are still problems just let me know.

**Comment by leecade** [comment]
@matthaias great job, I will use it in my project
